fortio_main.go

```diff
@@ -78,8 +78,10 @@ var (
 	goMaxProcsFlag    = flag.Int("gomaxprocs", 0, "Setting for runtime.GOMAXPROCS, <1 doesn't change the default")
 	profileFlag       = flag.String("profile", "", "write .cpu and .mem profiles to file")
 	grpcFlag          = flag.Bool("grpc", false, "Use GRPC (health check by default, add -ping for ping) for load testing")
-	grpcSecureFlag    = flag.Bool("grpc-secure", false, "Use secure transport (tls) for GRPC")
```

Comment:
that change isn't reflected in the README.md

see release/updateFlags.sh for how to update

we also need to document somewhere that https:// replaces -grpc-secure